client/src/partials/journal_voucher/journal_voucher.js

```diff
@@ -370,10 +382,9 @@ angular.module('bhima.controllers')
   // set the account for a row
   // remove all debtor/creditor properties that may
   // have been set on a previous selection
-  self.setAccount = function (row) {
+  vm.setAccount = function (row) {
     row.account_id = row.account.id;
     row.deb_cred_uuid = undefined;
```

Comment:
```undefined``` means the variable is declared but no value is assigned yet, to be semantically correct you should assign ```null``` which is an assignment value, and it is also a representation of no value.